share/completions/homectl.fish

```diff
@@ -0,0 +1,129 @@
+
+function __homectl_users
+    homectl list | string match -r -- '\S+\s+\d+\s+\d+' | string match -r -- '\S+' | string split0
+end
+
+function __homectl_subcommand_is
+    set -l cmd (commandline -poc)
+    if contains -- $cmd[-1] $argv
+        return 0
+    end
+    return 1
+end
+
+# Commands
+complete -c homectl -f
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a list -d 'List home areas'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a activate -d 'Activate a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a deactivate -d 'Deactivate a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a inspect -d 'Inspect a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a authenticate -d 'Authenticate a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a create -d 'Create a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a remove -d 'Remove a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a update -d 'Update a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a passwd -d 'Change password of a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a resize -d 'Resize a home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a lock -d 'Temporarily lock an active home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a unlock -d 'Unlock a temporarily locked home area'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a lock-all -d 'Lock all suitable home areas'
+complete -c homectl -n "not __fish_seen_subcommand_from $commands" -a with -d 'Run shell or command with access to a home area'
+complete -c homectl -n "__homectl_subcommand_is $commands_need_user" -xa ("__homectl_users")
+
+# Options
+complete -c homectl -s h -l help -d 'Show this help'
+complete -c homectl -l version -d 'Show package version'
+complete -c homectl -l no-pager -d 'Do not pipe output into a pager'
+complete -c homectl -l no-legend -d 'Do not show the headers and footers'
+complete -c homectl -l no-ask-password -d 'Do not ask for system passwords'
+complete -c homectl -s H -l host -d 'Operate on remote host'
+complete -c homectl -s M -l machine -d 'Operate on local container'
+complete -c homectl -l identity -d 'Read JSON identity from file' -r
+complete -c homectl -l json -d 'Output inspection data in JSON' -xa 'pretty short off'
+complete -c homectl -s j -d 'Equivalent to --json (on TTY) or short (otherwise)'
+complete -c homectl -l export-format -d 'Strip JSON inspection data' -xa 'full stripped minimal'
+complete -c homectl -s E -d 'Equals -j --export-format (twice to minimal)'
+complete -c homectl -o EE -d 'Equals -j --export-format=minimal'
+
+# User Record Properties
+for condition in '__fish_seen_subcommand_from create update'
```

Comment:
The loop seems unnecessary since it's only executed once.
Maybe instead use `set -l condition '__fish_seen_subcommand_from create update'` ?